products/controlm.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Control-M
```

Comment:
Does this page document the Control-M/Server for UNIX and Microsoft Windows product whose dates are available on https://webapps.bmc.com/support/faces/az/prodallversions.jsp?seqid=122988 ?

Reply:
I am asking because there are many Control-M product on https://webapps.bmc.com/support/faces/az/supportlisting.jsp.

Reply:
The one I had in mind when I wrote this was Control-M/Agent for UNIX and Microsoft Windows, but the support dates are shared by the software you mentioned and by Control-M/Enterprise Manager, by Control-M/Workflow Insights and every other one I checked, with the exception of the Control-M SaaS agent, and Control-M/Enterprise Manager PW seems to be missing some older support dates but more recent versions follow the same support dates.  (Can't tell whether this is intentional or not). So I am tempted to keep it generic and maybe we could just add some wording in for exceptions or it would be fairly easy to do separate pages for the ones that don't follow the same support dates (currently two products).

Reply:
See https://github.com/endoflife-date/endoflife.date/pull/5118#discussion_r1712934539.